ruby-challenges.rb

```diff
@@ -19,13 +27,43 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
-
+def sum_array(array)
+  puts array.sum
+end
+puts sum_array(nums_array1)
+puts sum_array(nums_array2)
```

Comment:
`puts` is a great tool for development. But once your code is working, it is a good idea to only have the log on the method call.